docs/UsersGuide.asciidoc

```diff
@@ -157,65 +398,163 @@ the same page of the table.
+longer history. This applies to both the index dashboard and group
+overview page.
+
+Example to show up to three week old builds instead of the default two
+weeks with up to 20 builds instead of up to 10 being the default for the
+group overview page:
+
+....
+http://openqa/group_overview/1?time_limit_days=21&limit_builds=20
+....
+
+
+=== Web UI controls to filter only tagged or all builds https://github.com/os-autoinst/openQA/pull/807[gh#807]
+
+Using a new query parameter `only_tagged=[0|1]' the list can be
+filtered, e.g. show only tagged (important) builds.
+
+Example screenshot:
+
+image::images/job_group-limit_builds_tagged.png[Show only tagged or all builds]
+
+Related issue: https://progress.opensuse.org/issues/11052[#11052]
+
+
+=== Carry over bugrefs from previous jobs in same scenario if still failing https://github.com/os-autoinst/openQA/pull/564[gh#564]
+
+It is possible to label all failing tests but tedious to do by a human
+user as many failures are just having the same issue until it gets
+fixed. It helps if a label is preserved for a build that is still
+failing. This idea is inspired by
+https://wiki.jenkins-ci.org/display/JENKINS/Claim+plugin and has been
+activated for bugrefs.
+
+Does not carry over bugrefs over passes: After a job passed a new issue
+in a subsequent fail is assumed to be failed for a different reason.
+
+Related issue: https://progress.opensuse.org/issues/10212[#10212]
+
+
+=== Pinning comments as group description
+
+This is possible by adding the keyword `pinned-description` anywhere in
+a comment on the group overview page. Then the comment will be shown at
+the top of the group overview page. However, it only works as operator
+or admin.
+
+
+=== Developer mode ===
+
+The developer mode allows to:
+
+* Create or update needles from +assert_screen+ mismatches ("re-needling")
+* Pause the test execution (at a certain module) for manual investigation of the SUT
+
+It can be accessed via the "Live View" tab of a running test. Only registered
+users can take control over a tests. Basic instructions and buttons providing further
+information about the different options are already contained on the web page itself.
+So I am not repeating that information here and rather explain the overall workflow.
+
+In case the developer mode in not working on your instance, try to follow the
+<<Pitfalls.asciidoc#debugdevelmode,steps for debugging the developer mode under 'Pitfalls'>>.
+
+==== Workflow for creating or updating needles ====
+
+1. In case a new needles should be created, add the corresponding +assert_screen+s to
+   your test.
+2. Start the test with the +assert_screen+s which are supposed to fail.
+3. Select "Pause on +assert_screen+ failure and confirm.
+4. Wait until the test has paused. There is a button to skip the current timeout to speed
+   this up.
+5. A button for acessing the needle editor should occour. It may take a few seconds till
```

Comment:
occour -> occur

Reply:
I'm just moving the section with your typo, you know that, right? ;) Fixed, updated

Reply:
I know